app/models/school_device_allocation.rb

```diff
@@ -88,6 +88,14 @@ def raw_allocation
     self[:allocation]
   end
 
+  def devices_available_to_order
+    if has_virtual_cap_feature_flags? && is_in_virtual_cap_pool?
+      cap - devices_ordered
+    else
+      raw_cap - raw_devices_ordered
+    end
```

Comment:
```suggestion
    cap - devices_ordered
```

Won't `cap` and `devices_ordered` do this themselves?

Reply:
Doh!

Reply:
Done